### DISCUSSION TITLE
Getting "Login attempt or request with invalid authentication" warnings in Home Assistant

### DISCUSSION THREAD

**Discussion by lexam79** [music-assistant/support#2824] [Q&A] [answered]
Hi all,
I'm running the latest beta (2.3.0b7) as a docker container. Also running Home Assistant (2024.8.3), again in docker, on the same host. Everything seems to work fine, however, whenever I restart Music Assistant (and every few hours anyway), I get a warning in Home Assistant like the following:

```
Logger: homeassistant.components.http.ban
Source: components/http/ban.py:135
integration: HTTP ([documentation](https://www.home-assistant.io/integrations/http), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+http%22))
First occurred: 4:07:56 PM (4 occurrences)
Last logged: 4:49:58 PM

Login attempt or request with invalid authentication from DOCKER_HOST_IP_ADDRESS (DOCKER_HOST_IP_ADDRESS). Requested URL: '/media/local/An%20Epic%20Story.mp3?authSig=REDACTED_TOKEN_HERE'. (Lavf/60.3.100)
```

Funnily enough, none of the players in MA isn't - currently - trying to play that specific file. I must have tried playing it at some point in the past, but I can't remember. It's not causing any problems, but it's driving me crazy and I can't seem to find more information about it, like which player is trying to play the file.

Has anyone seen anything similar?

Thank you

**Comment by lexam79** [comment]
Noone? 🤔

**Reply by OzGav** [reply]
I would say no one. Haven’t heard of this before?

**Comment by lexam79** [comment]
OK, I've narrowed it down a bit, to reloading the "Music Assistant" music provider. Whenever I reload that, it triggers the warning in Home Assistant...

**Reply by OzGav** [reply]
@marcelveldt ?

**Comment by lexam79** [comment]
OK, nailed it. As soon as I disable "Recently played tracks" in "Music Assistant" provider, it stops happening. So it seems that at some point in the past I played that track and Music Assistant is still trying to reference that track in the recently played playlist, using the same - now invalid - authorization token.

**Reply by OzGav** [reply]
Why do you say “that track”. There should be lots of tracks in that playlist…

**Reply by lexam79** [reply]
There are indeed multiple tracks, from various providers. However, the only one ever referenced in the login failure message is a specific track (an mp3 file which is uploaded in the media folder of Home Assistant).

**Reply by marcelveldt** [reply, marked as answer]
Yes, this issue is fixed a while ago so these entries do not get stored in the db anymore but existing entries are not cleaned up. For now either initiate a full library rebuild or wait for a patch

**Reply by lexam79** [reply]
Thank you, I've reset my library.
